src/structuring_element.jl

```diff
@@ -0,0 +1,356 @@
+abstract type MorphologySE{N} end
+
+"""
+    SEMask{N}()
+
+The N-dimensional structuring element in terms of bool array as mask. Typically, `true`
+represents the foreground and `false` represents the background.
```

Comment:
From this description it's not entirely clear what this is for. Is it a trait? Perhaps a usage example would help?